en/components/general/cli-overview.md

```diff
@@ -29,29 +47,21 @@ ig
     <p style="text-align:center;">Building Your First Ignite UI CLI App</p>
 </div>
 
-#### Using commands
-To create new project, add a component template and build it use the following Ignite UI CLI [commands](#commands):
-```cmd
-ig new <project name> --framework=angular --type=igx-ts 
-cd <project name>
-ig add <component/template> <component_name>
-ig start 
-```
-
-To execute the above commands, you should replace the *project name* and *component_name* placeholders with an actual value.
-After running these commands, your new project will be built, served and automatically opened in your default browser.
+Otherwise you can use the Ignite UI Schematics collection or CLI commands for creating an Ignite UI project, adding new component/s or building the application.
```

Comment:
Ignite UI **for Angular**, **Ignite UI** CLI.
Ignite UI Schematics might as well refer to jQuery schematics, CLI commands are used in both cases so it's not really an "or". Also the schematics definitely don't do building, and the `ig build` is merely a proxy, so meh.
I'd just remove this one.
```suggestion
```